File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/LinePath.java
package ja.burhanrashid52.photoeditor;

import android.graphics.Paint;
import android.graphics.Path;

class LinePath {
    private final Paint mDrawPaint;
    private final Path mDrawPath;

    LinePath(final Path drawPath, final Paint drawPaints) {
        mDrawPaint = new Paint(drawPaints);
        mDrawPath = new Path(drawPath);
    }

    Paint getDrawPaint() {
        return mDrawPaint;
    }

    Path getDrawPath() {
        return mDrawPath;
    }
}

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/LinePath.kt
package ja.burhanrashid52.photoeditor

import android.graphics.Paint
import android.graphics.Path

internal class LinePath(drawPath: Path?, drawPaints: Paint?) {
    val drawPaint: Paint
    val drawPath: Path

    init {
        drawPaint = Paint(drawPaints)
        this.drawPath = Path(drawPath)
    }
}